src/UnixConnector.php

```diff
@@ -31,8 +31,9 @@ public function connect($path)
             return Promise\reject(new InvalidArgumentException('Given URI "' . $path . '" is invalid'));
         }
 
-        if (strlen($path) > 104) {
-            return Promise\reject(new InvalidArgumentException('Given URI "' . $path . '" exceeds maximum allowed length of 104 bytes'));
+        $maxLen = PHP_OS === 'Darwin' ? 104 : 108;
```

Comment:
Is the max-length including or excluding the trailing NUL byte? A link would probably be good for future reference.